src/components/charts/common-chart-options.js

```diff
@@ -1,6 +1,6 @@
 /* this contains global options for all charts */
-class ChartOptions {
-    constructor(theme, title, type) {
+class CommonChartOptions {
+    constructor() {
```

Comment:
We no longer need to pass in the type of chart or the theme, as any options that need to check these have moved to `SpecificChartOptions`